### DISCUSSION TITLE
Rappel des bonnes pratiques d'ecoconception

### DISCUSSION THREAD

**Discussion by Trankilloman** [cnumr/best-practices#88] [General]
Comme indiqué [ici](https://github.com/cnumr/best-practices/discussions/59#discussioncomment-1534909), je pense qu'il serait nécessaire de rappeler, a plusieurs endroits dans le livre, les bonnes pratiques général d'ecoconception, a savoir, notamment, le fait qu'il faut éviter de prendre chaque bonne pratique comme solution a appliquer obligatoirement mais bien, réfléchir à la pertinence en fonction de son unité fonctionnelle, de l'objectif de sa démarche, de son périmètre...
Par extension, je pense qu'il faut éviter les termes du type "supprimer", "bannir" pour les remplacer par "éviter" ou d'autres verbes moins directifs.
Rappeler également qu'il est à éviter de comparer 2 sites ayant des fonctions différentes serait une bonne chose.

**Comment by ACTLEM** [comment]
Merci @Trankilloman, tu as 100% raison pour les termes très directifs 👍
